docs/sources/articles/configuring.md

```diff
@@ -60,39 +80,158 @@ You can start/stop/restart `docker` using
 ### Configuring Docker
 
 You configure the `docker` daemon in the `/etc/default/docker` file on your
-system.  You do this by specifying values in a `DOCKER_OPTS` variable. 
+system.  You do this by specifying values in a `DOCKER_OPTS` variable.
 To configure Docker options:
 
-1. Log into your system as a user with `sudo` or `root` privileges.
+1. Log into your host as a user with `sudo` or `root` privileges.
 
-2. If you don't have one, create the `/etc/default/docker` file in your system. 
-
-	Depending on how you installed Docker, you may already have this file.
+2. If you don't have one, create the `/etc/default/docker` file on your host. Depending on how
+you installed Docker, you may already have this file.
 
 3. Open the file with your favorite editor.
 
-		$ sudo vi /etc/default/docker
-		
+    ```
+    $ sudo vi /etc/default/docker
+    ```
```

Comment:
Why did you change all these code blocks to use back-ticks? Indenting with 4 or 8 spaces should work (4 or 8 spaces depend on the "nesting" level; if you're in a list or not).

I know the "triple back-ticks" are valid, and do the same, but we use the indenting (almost) anywhere.